adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/Internal/ClientCreds/JsonWebToken.cs

```diff
@@ -115,9 +115,9 @@ private static string UrlEncodeSegment(byte[] segment)
             return Base64UrlEncoder.Encode(segment);
         }
 
-        private static string EncodeHeaderToJson(IClientAssertionCertificate credential)
+        private static string EncodeHeaderToJson(IClientAssertionCertificate credential, bool sendX5C)
```

Comment:
sendX5C name does not reflect usage of parameter

Reply:
@trwalke

---
In reply to: [202846565](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1124#discussion_r202846565) [](ancestors = 202846565)

Reply:
Updated parameter to "sendCertificate"